src/pages/index.tsx

```diff
@@ -1,12 +1,9 @@
-import { signIn, signOut, useSession } from "next-auth/react";
+import { signIn, useSession } from "next-auth/react";
 import Head from "next/head";
-import Link from "next/link";
 import { Button } from "@/components/ui/button";
-import { api } from "@/utils/api";
 import { Outline } from "@/components/Outline";
 
 export default function Home() {
-  // const hello = api.post.hello.useQuery({ text: "from tRPC" });
   const { data: sessionData } = useSession();
 
   if (!sessionData) {
```

Comment:
one thing you can do, and even create a backburner issue for is to create some kind of standardized head component so you can give each page a title. This will let users see Finance Tracker | Expenses or something similar in their tab so they have more specificity given several tabs or looking at their history. I can show you an example if you want.